toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -22,6 +29,22 @@ func doCustomizations(baseConfigPath string, config *imagecustomizerapi.SystemCo
 	return nil
 }
 
+func updateHostname(hostname string, imageChroot *safechroot.Chroot) error {
```

Comment:
Not sure if you ever expect to run tools via `shell` in this context but in our full installer we update the shell's environment to reflect the new name as well. (`installutils.go:updateHostname()`).

Reply:
Yeah, I saw that. I decided to omit it until I knew what it was being used for.